frontend/src/chat/components/ChatWindow.vue

```diff
@@ -203,6 +227,14 @@ onMounted(() => {
     subtree: true,
   };
   observer.observe(document.getElementById("chatInput"), config);
+
```

Comment:
This is also the fix for pasting stuff in chad.
This removes unwanted formatting :D